api/v1beta1/ibporderer.go

```diff
@@ -236,27 +236,11 @@ func (s *IBPOrdererSpec) DomainSet() bool {
 }
 
 func (s *IBPOrdererSpec) IsPrecreateOrderer() bool {
-	if s.IsPrecreate == nil {
-		return false
-	}
-
-	if *s.IsPrecreate == BoolTrue {
-		return true
-	}
-
-	return false
+	return s.IsPrecreate != nil && *s.IsPrecreate
```

Comment:
This line is great - much clearer.  Thanks for letting us know about the `pointer` package! 

This line is much cleaner than the old function.  No need to update the PR, but I wonder if using the BoolDeref() here would make the intent even clearer.  E.g.: 

```
return pointer.BoolDeref(s.IsPrecreate, false)
```

It would be really good to establish a common pattern for comparisons with maybe-null boolean attributes, and carry this forward.